docs/guides/prod-guide.md

```diff
@@ -0,0 +1,80 @@
+
+nat_gateway: single # for large clusters making requests to services outside the cluster (e.g. S3 or database) use highly_available
+```
+
+You can make your loadbalancers private to prevent your APIs from being publicly access. In order to access your APIs, you will need to setup VPC peering between the Cortex cluster's VPC and the VPC containing the consumers of the Cortex APIs. See the [VPC peering guide](./vpc-peering.md) for more details.
+
+```yaml
+api_load_balancer_scheme: internal
+```
+
+You can also restrict access to your loadbalancers by IP address to improve security even further. This can be done if the API loadbalancer is public or private.
+
+```yaml
+api_load_balancer_cidr_white_list: [0.0.0.0/0]
+```
+
+### Ensure node provisioning
+
+You can get the cost savings of spot instances and the reliability of on-demand instances by taking advantage of the priority field in nodegroups. You can deploy two nodegroups, one that is spot and another that is on-demand. Set the priority of the spot nodegroup to be higher than the priority of the on-demand nodegroup. This encourages the cluster-autoscaler to try to spin up instances from the spot nodegroup first. If after 5 minutes, the spot nodegroup is unable to scale up because there are no more spot instances available, the on-demand nodegroup  will be used instead.
+
+```yaml
+node_groups:
+  - name: gpu-spit
+    instance_type: g4dn.xlarge
+    min_instances: 0
+    max_instances: 5
+    spot: true
+    priority: 100
+  - name: gpu-on-demand
+    instance_type: g4dn.xlarge
+    min_instances: 0
+    max_instances: 5
+    priority: 1
+```
+
+### Considerations for large clusters
+
+If you plan on spinning up a Cortex cluster reaching 400 nodes or a 1000 pods, you might want to consider setting `prometheus_instance_type: ` to a larger instance type. A good rule of thumb is a t3.medium instance can reliably handle 400 nodes and 800 pods.
+
+## API Spec
+
+### Container design
+
+Configure your healthchecks to be as accurate as possible to mitigate the chances of requests being routed to pods that are not ready yet.
+
+### Pods section
+
+Make sure that `max_concurrency` is set to match the concurrency supported by your container.
+
+Tune `max_queue_length` to lower values if you would like to more aggressively redistribute requests to newer pods as your API scales up rather than allowing requests to linger in queues. This would mean that the clients consuming your APIs should implement a retry logic with a delay (such as exponential backoff).
+
+### Compute section
+
+Make sure to specify all of the relevant compute resources, especially the cpu and memory to ensure that your pods aren't starved for resources.
+
+### Autoscaling
+
+Revisit the autoscaling docs for your [Realtime APIs](../workloads/realtime/autoscaling.md) and [Async APIs](../workloads/async/autoscaling.md) to effectively handle your production traffic by tuning the sensitivity, rate of autoscaling and over-provisioning.
```

Comment:
Revisit the autoscaling docs for your [Realtime APIs](../workloads/realtime/autoscaling.md) and [Async APIs](../workloads/async/autoscaling.md) to effectively handle your production traffic by tuning the sensitivity, rate, and over-provisioning of the autoscaler.